masterlist.yaml

```
@@ -14276,6 +14276,12 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/21625/' ]
     msg: [ *patch3rdParty_SRPC_Redux ]
 
+  - name: 'dwagonbown goes uwu.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/35441/']
+    group: 'Late Loaders'
```

Anchors should be used when available:
```suggestion
    group: *lateLoadersGroup
```